spectral_cube/tests/test_casafuncs.py

```diff
@@ -68,6 +68,43 @@ def make_casa_testimage(infile, outname):
         ia.close()
         ia.done()
 
+def make_casa_stokes_testimage(infile, outname):
+
+    infile = str(infile)
+    outname = str(outname)
+
+    if not CASA_INSTALLED:
+        raise Exception("Attempted to make a CASA test image in a non-CASA "
+                        "environment")
```

Comment:
do we not have a decorator for this?

Reply:
I don't think we have one for the tests.